src/main/java/com/celements/pagelayout/LayoutScriptService.java

```diff
@@ -96,8 +96,8 @@ public boolean deleteLayout(String layoutSpaceName) {
   }
 
   public PageLayoutApi getPageLayoutApiForRef(SpaceReference layoutSpaceRef) {
-    if (layoutExists(layoutSpaceRef)) {
-      return new PageLayoutApi(layoutSpaceRef, getContext());
+    if (layoutSpaceRef != null) {
```

Comment:
script services must cope with nullable values. layoutExists is not nullable thus please keep the null check additonaly before the layoutExists check.